develop/networking.md

```diff
@@ -26,69 +26,28 @@ so this page is split up into a few categories.
 
 ## Why Is Networking Important? {#why-is-networking-important}
 
-The importance of networking can be shown by a simple code example.
+Packets of data are the core concept of networking in Minecraft. Packets are made up of arbitrary data that can be sent either from
+the server to the client or from the client to the server. Almost everything you do in the game involves networking in some way, whether
+you know it or not. For example, when you jump in the game, a packet is sent to the server to tell it that you jumped. The server then
+sends a packet to all the other clients to tell them that you jumped.
```

Comment:
I'm not sure how much it matters, but this doesn't sound factually correct 😅 AFAIK a jump action (and other forms of player movement) just gets boiled down to a change in position and velocity

Reply:
Thats true, ill change this to a chat message. Just wanted something simple to get the idea across.